srcts/src/components/sidebar.ts

```diff
@@ -242,6 +266,311 @@ class Sidebar {
     containers.forEach((container) => new Sidebar(container as HTMLElement));
   }
 
+  /**
+   * Check if the sidebar should be resizable in the current state.
+   * @private
+   * @returns {boolean}
+   */
+  private _shouldEnableResize(): boolean {
```

Comment:
I'd find it more readable if this was ordered towards the bottom of these methods